lektor/packages.py

```diff
@@ -292,7 +332,10 @@ def update_cache(package_root, remote_packages, local_package_path, refresh=Fals
                     package_root, os.path.join(local_package_path, package[1:])
                 )
             else:
-                download_and_install_package(package_root, package, version)
+                download_and_install_packages(
+                    package_root,
+                    ["%s%s%s" % (package, version and "==" or "", version or "")],
```

Comment:
I know you just moved this code from where it used to be, but I'd rather see this as:
```python
requirement_spec = f"{package}=={version}" if version else package
download_and_install_packages(package_root, [requirement_spec])
```

(Perhaps a full `if` ... `else` ... to compute `requirement_spec` would be even more readable.)

Reply:
I think a full if:else doesn't add much readability. Still I would really like to keep this patch somewhat minimal, given that it might become obsolete soon anyway.